start.sh

```diff
@@ -7,4 +7,4 @@ chmod go-w filebeat/zeek.yml
 docker-compose build
```

Comment:
I deliberately chose to not detach so I could view the logs, and would prefer to keep it that way for this. Maybe adding it as a flag to `start.sh` would be a better option?